thresholdkey_generator.go

```diff
@@ -19,7 +20,7 @@ import (
 //               with Applications to Electronic Voting
 //               Aarhus University, Dept. of Computer Science, BRICS
 type ThresholdKeyGenerator struct {
-	nbits                          int
+	publicKeyBitLength             int
```

Comment:
Shouldn't it be public same as other fields in this struct?

Reply:
Yeah, it can be public. On the other hand, there is no point for `Random` to be public. I'll update it.